ouroboros-consensus/src/Ouroboros/Consensus/Node/ProtocolInfo/Byron.hs

```diff
@@ -11,6 +11,7 @@ module Ouroboros.Consensus.Node.ProtocolInfo.Byron (
     protocolInfoByron
   , ByronConfig
   , PBftSignatureThreshold(..)
+  , defaultPBftSignatureThreshold
```

Comment:
Not related to this PR, I guess but any idea why this name is capitalized as `PBft`? Rather than `Pbft` or `PBFT`?

Reply:
PBft comes from Permissive BFT, and in Haskell we typically (I sin against this sometimes) capitalise acronyms when they are followed by another word to make it clear of which words a camelCase name consists. For example: `HTTPAPIPOST` vs. `HttpApiPost`, the latter is arguably easier to read.

In this case `PBFTSignatureThreshold` would only be slightly less readable.

And why not `Pbft` instead of `PBft`? Because we treat `Permissive` as a separate word? I'm not sure.